mon_school/www/sketches/index.html

```diff
@@ -52,3 +52,7 @@
   </ul>
 </nav>
 {% endmacro %}
+
+{% block script %}
+<script src="/assets/mon_school/js/sketch_like.js"></script>
```

Comment:
You can write an index.js file and that will be automatically included. It will be closer to the template and easier to find.

Reply:
The same script is also being used in the dedicated sketch page. So I named it according to its function.

Reply:
That's fine then. It is not a blocker for this, but it would be good to add the script to a bundle and include the bundle. 

I usually avoid to have JS files that does something just by including in the html. I prefer to have an explicit call from the template.

Reply:
Done